01-shapefile-attributes.Rmd

```diff
@@ -51,7 +52,7 @@ More on Packages in R - Adapted from Software Carpentry.</a>
 
 ##Data to Download
 
-Download the shapefiles neede to complete this lesson:
+Download the shapefiles needes to complete this lesson:
```

Comment:
Yes it does. Line 54/55 in lesson01.  Although I realized my fix is still a typo.  Argh.